alpine/src/main/scala/io/delta/alpine/internal/SnapshotImpl.scala

```diff
@@ -57,12 +54,10 @@ private[internal] class SnapshotImpl(
   def protocolScala: Protocol = state.protocol
   def metadataScala: Metadata = state.metadata
 
-  def setTransactions: Seq[SetTransaction] = state.setTransactions
```

Comment:
remove unused methods.

Reply:
wouldn't these be useful for testing? I'll add a TODO to remove them later, if they end up not being needed for tests

Reply:
We can also ignore `SetTransaction` in InMemoryLogReply. There is no place using it. We don't need `SetTransaction` until we add the write support.

Reply:
sounds good

Reply:
We might as well delete `CommitInfo` too then